autofit/graphical/utils.py

```diff
@@ -44,45 +128,44 @@ def __init__(self, dict_: Dict[Variable, Tuple[int, ...]]):
         super().__init__()
 
         self.update(dict_)
-        self.splits = np.cumsum([
-            np.prod(s) for s in self.values()], dtype=int)
+        self.splits = np.cumsum([np.prod(s) for s in self.values()], dtype=int)
         self.inds = [
-            slice(i0, i1) for i0, i1 in
+            slice(i0, i1)
+            for i0, i1 in
             # np.arange(i0, i1, dtype=int) for i0, i1 in
-            zip(np.r_[0, self.splits[:-1]], self.splits)]
-        self.sizes = {
-            k: np.prod(s, dtype=int) for k, s in self.items()}
+            zip(np.r_[0, self.splits[:-1]], self.splits)
+        ]
+        self.sizes = {k: np.prod(s, dtype=int) for k, s in self.items()}
 
     @classmethod
-    def from_arrays(cls, **arrays: Dict[str, np.ndarray]) -> "FlattenArrays":
-        return cls(**{k: np.shape(arr) for k, arr in arrays.items()})
+    def from_arrays(cls, arrays: Dict[str, np.ndarray]) -> "FlattenArrays":
+        return cls({k: np.shape(arr) for k, arr in arrays.items()})
 
     def flatten(self, arrays_dict: Dict[Variable, np.ndarray]) -> np.ndarray:
-        assert all(np.shape(arrays_dict[k]) == shape
-                   for k, shape in self.items())
-        return np.concatenate([
-            np.ravel(arrays_dict[k]) for k in self.keys()])
+        assert all(np.shape(arrays_dict[k]) == shape for k, shape in self.items())
+        return np.concatenate([np.ravel(arrays_dict[k]) for k in self.keys()])
 
     def unflatten(self, arr: np.ndarray, ndim=None) -> Dict[str, np.ndarray]:
         arr = np.asanyarray(arr)
         if ndim is None:
             ndim = arr.ndim
-        arrays = [
-            arr[(ind,) * ndim] for ind in self.inds]
+        arrays = [arr[(ind,) * ndim] for ind in self.inds]
         arr_shapes = [arr.shape[ndim:] for arr in arrays]
-        return {
-            k: arr.reshape(shape * ndim + arr_shape)
-            if shape or arr_shape else arr.item()
-            for (k, shape), arr_shape, arr in
-            zip(self.items(), arr_shapes, arrays)}
+        return VariableData(
+            {
```

Comment:
Given that VariableData only takes one argument I'd put the braces on the same line as the brackets